torch/linalg/__init__.py

```diff
@@ -813,6 +813,102 @@
     https://www.netlib.org/lapack/lug/node128.html
 """)
 
+ldl_factor_ex = _add_docstr(_linalg.linalg_ldl_factor_ex, r"""
+linalg.ldl_factor_ex(A, *, upper=False, hermitian=False, check_errors=False, out=None) -> (Tensor, Tensor, Tensor)
+
+Computes the LDL decomposition of a Hermitian/symmetric indefinite matrix.
+When :attr:`A` is complex valued it can be Hermitian (:attr:`hermitian` = `True`)
+or symmetric (:attr:`hermitian` = `False`).
+
+:attr:`upper` is a boolean value that controls whether the form of the decomposition is
+:math:`L D L^T` (:attr:`upper` = `False`) or :math:`U D U^T` (:attr:`upper` = `True`).
+If :attr:`hermitian` is `True` then transpose operation is the conjugate transpose.
+
+:math:`L` (or :math:`U`) and :math:`D` are stored in compact form in ``factors``.
+This format is used by :func:`torch.linalg.ldl_solve` for solving linear systems.
```

Comment:
Add a link to the LAPACK docs where the format is described. Same for the `info` below.

Reply:
I'm hesitant to use links to LAPACK documentation because URL doesn't look like something that persists in history and not changing between the releases:
https://www.netlib.org/lapack/explore-html/d3/db6/group__double_s_ycomputational_gad91bde1212277b3e909eb6af7f64858a.html

Reply:
Fair enough. Then perhaps say "the returned is that of LAPACK's sytrf" or so?